### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by spolti**
- Fix script to run on OSx
- Move .gitignore file to project's root directory

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by spolti** [comment]
> What's the part that breaks on MacOS?

$cmd &>>/dev/null

now it is here: https://github.com/opendatahub-io/caikit-tgis-serving/blob/main/demo/kserve/scripts/utils.sh#L158

need to rebase it, will do later...

or if you want to incorporate this little change in some of yours PRs, that's fine as well :)

**Comment by spolti** [comment]
> > $cmd &>>/dev/null
> 
> How is this breaking? It seems valid bash and zsh syntax to me

```
utils.sh: command substitution: line 136: syntax error near unexpected token `>'
utils.sh: command substitution: line 136: `$cmd &>>/dev/null'
```

**Comment by dtrifiro** [comment]
Oh, I forgot that MacOS ships with an ancient bash version (3.x) due to licensing issues. Can you rebase on main?

File: demo/kserve/scripts/install/kserve-install.sh [hunk 1]
```diff
@@ -14,9 +14,9 @@ then
   input="y"
 else
   echo "** Check User Workload Configmap for Kserve metrics before you execute this script **"
-  echo 
+  echo
+  # TODO point of improvement, execute the command and bring the real output from user's cluster.
```
Comment (dtrifiro) [inline demo/kserve/scripts/install/kserve-install.sh:18]: Is this required?

**Review by dtrifiro — CHANGES_REQUESTED** [review]

File: demo/kserve/scripts/install/kserve-install.sh (hunk 1 above)
Comment (spolti) [reply demo/kserve/scripts/install/kserve-install.sh]: No, this was edited by Jooho in his last PR.

**Pull request merged**